src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,60 @@
-export const Tabs = () => {};
+import React from 'react';
+import classNames from 'classnames';
+
+interface Tab {
+  id: string;
+  title: string;
+  content: string;
+}
+
+type Props = {
+  tabs: Tab[],
+  onTabSelected: (tab: Tab) => void;
+  selectedTabId: string;
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  onTabSelected,
+  selectedTabId,
+}) => {
+  const currentTab = tabs.find((tab) => (
+    tab.id === selectedTabId)) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map((tab) => {
+            const { id, title } = tab;
+            const selectedTab = id === currentTab.id;
```

Comment:
looks like a boolean variable to me, but is named just like I would name object 🤔
Imagine naming your dog Nazar. You will tell your friends about Nazar and they will think you're talking about a human, but it is actually a dog, so that name may easily confuse.

Naming matters!